t/discovery/eureka-weight.t

```diff
@@ -0,0 +1,168 @@
+                        headers={["Content-Type"] = "application/json",}
+              })
+            if not res then
+                ngx.say(err)
+                return
+            end
+            res, err = httpc:request_uri(uri, {
+                    method = "POST",
+                    body="{\"instance\":{\"instanceId\":\"two\",\"app\":\"APISIX-EUREKA-USER-SERVICE\",\"ipAddr\":\"127.0.0.1\",\"port\":{\"$\":1981,\"@enabled\":\"true\"},\"dataCenterInfo\":{\"@class\":\"com.netflix.appinfo.InstanceInfo$DefaultDataCenterInfo\",\"name\":\"MyOwn\"},\"hostName\":\"instance0.application0.com\",\"status\":\"UP\",\"metadata\":{\"weight\":\"80\"},\"overriddenStatus\":\"UNKNOWN\"}}",
+                    headers={["Content-Type"] = "application/json",}
+              })
+            if not res then
+                ngx.say(err)
+                return
+            end
+        }
+    }
+--- request
+GET /t
+--- response_body
+--- no_error_log
+[error]
+
+
+
+=== TEST 2: get APISIX-EUREKA-USER-SERVICE info from EUREKA
+--- yaml_config eval: $::yaml_config
+--- apisix_yaml
+routes:
+  -
+    uri: /eureka-weight/*
+    plugins:
+      proxy-rewrite:
+        regex_uri: ["^/eureka-weight/(.*)", "/${1}"]
+    upstream:
+      service_name: APISIX-EUREKA-USER-SERVICE
+      type: roundrobin
+#END
+--- request
+GET /eureka-weight/hello
+--- response_body
+hello world
+--- no_error_log
+[error]
+
+
+
+=== TEST 3: hit routes
+--- config
+    location /t {
+        content_by_lua_block {
+            local http = require "resty.http"
+            local uri = "http://127.0.0.1:" .. ngx.var.server_port
+                        .. "/eureka-weight/server_port"
+
+            local ports_count = {}
+            for i = 1, 10 do
+                local httpc = http.new()
+                local res, err = httpc:request_uri(uri, {method = "GET"})
+                if not res then
+                    ngx.say(err)
+                    return
+                end
+                ports_count[res.body] = (ports_count[res.body] or 0) + 1
+            end
+
+            local ports_arr = {}
+            for port, count in pairs(ports_count) do
+                table.insert(ports_arr, {port = port, count = count})
+            end
+
+            local function cmd(a, b)
+                return a.port > b.port
+            end
+            table.sort(ports_arr, cmd)
+
+            ngx.say(require("cjson").encode(ports_arr))
+            ngx.exit(200)
+        }
+    }
+--- request
+GET /t
+--- yaml_config eval: $::yaml_config
+--- response_body
+[{"count":8,"port":"1981"},{"count":2,"port":"1980"}]
+--- no_error_log
+[error]
+
+
+
```

Comment:
only one blank line is need.

Reply:
Test cases can run through the local test case, and fail to test on CI. Can you give me some advice

Reply:
> Test cases can run through the local test case, and fail to test on CI. Can you give me some advice

@qiujiayu need your help ^_^